libpkg/pkg_audit.c

```diff
@@ -340,8 +340,10 @@ pkg_audit_expand_entry(struct pkg_audit_entry *entry, struct pkg_audit_entry **h
 	LL_FOREACH(entry->packages, pcur) {
 		LL_FOREACH(pcur->names, ncur) {
 			n = calloc(1, sizeof(struct pkg_audit_entry));
-			if (n == NULL)
-				err(1, "calloc(audit_entry)");
+			if (n == NULL) {
```

Comment:
and here, ```err()``` terminates program execution with the given exit status; you've changed that to returning from this function.